src/hooks/export/cms_export_json_utilities.e

```diff
@@ -0,0 +1,35 @@
+note
+	description: "[
+			Usefull routines to export to JSON.
```

Comment:
Should we add this useful features into JSON lib, so no need to reinvent the wheel every time we need something similar.